src/constants/axis_placeable_objects.js

```diff
@@ -0,0 +1,30 @@
+/**
+* Copyright 2012-2020, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+module.exports = {
+    axisRefDescription: function(axisname, lower, upper) {
+        return [
+            'If set to a ', axisname, ' axis id (e.g. *', axisname, '* or',
+            '*', axisname, '2*), the `', axisname, '` position refers to a',
+            '', axisname, ' coordinate. If set to *paper*, the `', axisname, '`',
```

Comment:
There is no need to put an empty string at the start of this line (as well as some following lines).
So this could be simplified to
```js
axisname, ' coordinate. If set to *paper*, the `', axisname, '`',
```